bundle/jsky.app.ot/src/main/java/jsky/app/ot/viewer/SPElevationPlotPlugin.java

```diff
@@ -136,10 +136,6 @@ public void stateChanged(ChangeEvent e) {
     private void _addMenuItems(ElevationPlotMenuBar menuBar) {
         JMenu menu = menuBar.getViewMenu();
 
-        // need to disable manually adding and deleting targets, since
-        // it conflicts with the target lists in the observations
-        menu.remove(1);
-
         menu.add(_createViewLabelMenu());
```

Comment:
The target list menu item is always removed here, but it no longer needs to be removed because it is never added (see below).